### PR TITLE
update google analytics and some refs to 42go to use the kifi.com info

### PR DESCRIPTION
**Pull request by eishay**
@martinraison

### CODE REVIEW

File: kifi-backend/modules/sqldb/app/com/keepit/common/db/slick/DbRepo.scala [hunk 1]
```diff
@@ -74,9 +74,9 @@ trait DbRepo[M <: Model[M]] extends Repo[M] with FortyTwoGenericTypeMappers with
       case m: ModelWithState[M] if m.state == State[M]("inactive") => deleteCache(result)
       case _ => invalidateCache(result)
     }
-    if (changeListener.isDefined) session.onTransactionSuccess{
-     if(newItem) changeListener.get(RepoEntryAdded(result))
-     else changeListener.get(RepoEntryUpdated(result))
+    changeListener.isDefined) session.onTransactionSuccess{
```
Comment (martinraison) [inline kifi-backend/modules/sqldb/app/com/keepit/common/db/slick/DbRepo.scala]: ? might be a bad keystroke

**Comment by martinraison** [comment]
see comment - thanks for doing this!

File: kifi-backend/modules/sqldb/app/com/keepit/common/db/slick/DbRepo.scala (hunk 1 above)
Comment (eishay) [inline kifi-backend/modules/sqldb/app/com/keepit/common/db/slick/DbRepo.scala]: yes, let me fix, thanks!

File: kifi-backend/modules/sqldb/app/com/keepit/common/db/slick/DbRepo.scala [hunk 2]
```diff
@@ -74,9 +74,10 @@ trait DbRepo[M <: Model[M]] extends Repo[M] with FortyTwoGenericTypeMappers with
       case m: ModelWithState[M] if m.state == State[M]("inactive") => deleteCache(result)
       case _ => invalidateCache(result)
     }
+
     if (changeListener.isDefined) session.onTransactionSuccess{
-     if(newItem) changeListener.get(RepoEntryAdded(result))
-     else changeListener.get(RepoEntryUpdated(result))
+       if(newItem) changeListener.get(RepoEntryAdded(result))
+       else changeListener.get(RepoEntryUpdated(result))
```
Comment (2is10) [inline kifi-backend/modules/sqldb/app/com/keepit/common/db/slick/DbRepo.scala:80]: looks like indentation is going from 1 space to 3 spaces here. should be 2.

**Pull request merged**